.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+node_js:
+ - 'node'
```

Comment:
We should explore using a `lts` version of node here as the current version of Node (`v10.1.0`) broke some of my older projects - food for thought!

Reply:
Good thought.  We'll use `lts/*`